sumcheck/src/prover/high_degree.rs

```diff
@@ -458,7 +458,7 @@ fn sumcheck_round<E: FieldElement>(
                     &mut numerators,
                     &mut denominators,
                 );
-                poly_evals[0] = evaluate_composition_poly(
+                poly_evals[0] += evaluate_composition_poly(
```

Comment:
Was this a bug previously?

Reply:
This is part of #331, which explains why it was not caught